docs/DafnyRef/UserGuide.md

```diff
@@ -108,7 +108,7 @@ identified with the name `lib` on the command line. For example, to build multip
 Dafny files into a single build artifact for shared reuse, the command would look something like:
 
 ```bash
-dafny build -t:lib A.dfy B.dfy C.dfy --out MyLib.doo
+dafny build -t:lib A.dfy B.dfy C.dfy --output:MyLib.doo
```

Comment:
Yes it can and does. In this case it would need to be marked with  %check-cli and it would need to be a valid command - that is, A.dfy etc. would need to exist.